dev-env-setup.md

```diff
@@ -0,0 +1,59 @@
+# Set Up Your Starport Development Environment 
+
+To ensure you have a successful experience developing with Starport, make sure that your local system meets these technical requirements.
+
+Starport is supported for the following operating systems:
+
+- GNU/Linux
+- macOS
+- Windows Subsystem for Linux (WSL)
+
+## Install Go
+
+This installation method removes existing Go installations, installs Go in `/usr/local/go/bin/go`, and sets the environment variables.
+
+1. Install [Go](https://golang.org/doc/install) (**version 1.16** or higher).
+
+2. Download the binary release that is suitable for your system.
```

Comment:
```suggestion
2. Download the release that is suitable for your system.
```

Because for some systems it's a binary and for some it's a tarball (manual installation).